workspace/utilities/lib/heading.xsl

```
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- CORE-ELEMENT: heading ======================================================================-->
+	<xsl:template name="heading">
+		<xsl:param name="level" select="'1'"/>
```

We should throw an exception is anything else than 1,2,3,4,5,6 is passed